Bottom: 2b31961be768ba1fa92add7767840a4ef0ce43b1
Top:    1dea80a703036b8141e234ea76f2cf07059cf89f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:38:16 +0000

Refresh of configure-ac-save-and-restore-

---

diff --git a/configure.ac b/configure.ac
index 46af266..8ed21a0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -704,6 +704,7 @@ for e in $HIDLIST; do
     case $e in
       lesstif )
 	AC_PATH_XTRA
+	save_CPPFLAGS="$CPPFLAGS"
 	CPPFLAGS="$X_CFLAGS"
 	AC_CHECK_LIB(X11, XOpenDisplay, , , $X_LIBS)
 	AC_CHECK_LIB(ICE, main, , , $X_LIBS)
@@ -712,7 +713,8 @@ for e in $HIDLIST; do
 	AC_CHECK_LIB(Xt, XtOpenDisplay, , , $X_LIBS)
 	AC_CHECK_LIB(Xmu, main, , , $X_LIBS)
 	AC_CHECK_LIB(Xpm, main, , , $X_LIBS)
-        AC_CHECK_LIB(Xm, XmCreateMainWindow, , , $X_LIBS)
+	AC_CHECK_LIB(Xm, XmCreateMainWindow, , , $X_LIBS)
+	CPPFLAGS="$save_CPPFLAGS"
 	case $ac_cv_lib_Xm_XmCreateMainWindow in
 	  no )
 	    AC_MSG_ERROR([You don't seem to have the Lesstif development environment installed.])
